packages/cozy-client/src/models/contact.spec.js

```diff
@@ -421,3 +428,65 @@ describe('getIndexByFamilyNameGivenNameEmailCozyUrl', () => {
     expect(result).toEqual('john.smith@cozy.cc')
   })
 })
+
+describe('getPrimaryCozyDomain', () => {
+  it('should returns empty string if no cozy url', () => {
+    const contact = {
+      cozy: [{ url: '' }]
+    }
+    const expected = ''
+
+    const result = getPrimaryCozyDomain(contact)
+    expect(result).toEqual(expected)
+  })
+
+  it('should returns cozy url without "https://"', () => {
+    const contact = {
+      cozy: [{ url: 'https://smith.mycozy.cloud' }]
+    }
+    const expected = 'smith.mycozy.cloud'
+
+    const result = getPrimaryCozyDomain(contact)
+    expect(result).toEqual(expected)
+  })
+
+  it('should returns cozy url without "http://"', () => {
+    const contact = {
+      cozy: [{ url: 'http://smith.mycozy.cloud' }]
+    }
+    const expected = 'smith.mycozy.cloud'
+
+    const result = getPrimaryCozyDomain(contact)
+    expect(result).toEqual(expected)
+  })
+
+  it('should returns cozy url with pathname if any', () => {
+    const contact = {
+      cozy: [{ url: 'https://smith.mycozy.cloud/contacts' }]
+    }
+    const expected = 'smith.mycozy.cloud/contacts'
```

Comment:
Since this method is in CC/models/Contact, I'm expecting to be really true. If we have something in the pathname, this is not part on the "CozyDomain". So we should not return it